src/main/resources/catalog.json

```diff
@@ -409,5 +428,47 @@
     "aliases" : [ ],
     "constraints" : [ ],
     "catalogId" : "DEFAULT"
+  }, {
+    "id" : 13,
+    "name" : "fileName",
+    "displayName" : "File name",
+    "description" : "File name",
+    "dataType" : "STRING",
+    "indexed" : false,
+    "sortable" : true,
+    "required" : false,
+    "requestContext" : false,
+    "examples" : null,
+    "aliases" : [ ],
+    "constraints" : [ ],
+    "catalogId" : "DEFAULT"
+  }, {
+    "id" : 14,
+    "name" : "usages",
+    "displayName" : "Usages",
+    "description" : "Usage of the entity",
+    "dataType" : "STRING",
```

Comment:
This sounds like it should be some sort of array instead of string unless you make it `usage` and log an audit event for each usage. This could be simply a naming issue.

Reply:
If it is creating a fingerprint for every usage, will this not always be a string? getUsages returns a HashTable<String, RangeSet>. Can you please help me understand this?

Reply:
As discussed in our chat, the keys will correspond to a psuedo filename for a credentials id and the name, and the RangeSet is the set of build numbers the credential was used in.